pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -1203,10 +1188,26 @@ func (lbaas *LbaasV2) EnsureLoadBalancerDeleted(clusterName string, service *v1.
 	return nil
 }
 
+func (lbaas *LbaasV2) IsNotFound(err error) bool {
```

Comment:
Just move this into the real `isNotFound()` - in fact I thought I had already extended that to check `ErrNotFound` some time ago, but apparently not...